WebAppConfig.ps1

```diff
@@ -0,0 +1,67 @@
+param(
```

Comment:
Please add `$ErrorActionPreference = "Stop"` and `Set-StrictMode -Version Latest` to the top

Reply:
I will add these settings, thanks for the tip. I'm not very experienced with PS and did not know of them...